frontend/src/components/events/index.jsx

```diff
@@ -18,44 +16,46 @@ class Events extends Component {
     this.props.fetchEvents();
   }
 
-  selectForEdit = id => {
-    let edit_event = this.props.events[id];
-    this.setState({
-      event: edit_event
-    });
-  };
-
   render() {
-    return (
-      <div>
-        <h1>Upcoming Events</h1>
-        <div className="eventbright-results">
-          <EventList>
-            <CardDeck>
-              {this.props.events.map(event => {
-                return (
-                  <EventItems
-                    key={event.id}
-                    name={
-                      event.name
-                        ? event.name.text
-                        : "Young Masterbuilders In Motion Event"
-                    }
-                    description={
-                      event.description ? event.description.text : ""
-                    }
-                    start={event.start ? event.start.local : "TBA"}
-                    end={event.end ? event.end.local : "TBA"}
-                    logo={event.logo ? event.logo.url : logo}
-                    url={event.url}
-                  />
-                );
-              })}
-            </CardDeck>
-          </EventList>
-        </div>
-      </div>
-    );
+    if (this.props.events.length) {
```

Comment:
to handle the logic down in EventCard we can do 

```
let events = this.props.events.filter((s) => ['live', 'started', 'ended', 'completed'].includes(s.status))
if (events.length) {
...
}
```

will have to update all the `this.props.events` to `just events`